drivers/SmartThings/zigbee-thermostat/profiles/thermostat-popp-danfoss.yml

```diff
@@ -4,24 +4,29 @@ components:
   capabilities:
   - id: thermostatMode
     version: 1
+    config:
+      - key: "thermostatMode.value"
+        enabledValues:
+        - heat
```

Comment:
```suggestion
    config:
      values:
        - key: "thermostatMode.value"
          enabledValues:
          - heat
        - key: "{{enumCommands}}"
          enabledValues:
          - heat
```

Reply:
There was a missing "value" field:
```
  - id: thermostatMode
    version: 1
    config:
      values:
        - key: "thermostatMode.value"
          enabledValues:
          - heat
 ```

Reply:
Wait, I missed the 
```
- key: "{{enumCommands}}"
          enabledValues:
          - heat
 ```
is that also required?

Reply:
It looks like the thermoStateMode capability does not need the additional "{{enumCommands}}" config because it has a setter method as explained here: https://smartthings.atlassian.net/wiki/spaces/~bflorian/pages/2462811346/Embedded+Device+Configurations#Enum-commands-and-attribute-values

Reply:
@ctowns If the setter method `key: setThermostatMode` can be used instead of `key: "{{enumCommands}}"` then ok. But it has to be defined here anyway.
I recommend to double-check it (the UI) with the device/virtual device.

Reply:
Thank you for the help! I also confirmed with Bob that the setter method `key: setThermostatMode` is the correct config to use here.